### PR TITLE
Clarify project proposal and approval requirements

### PR DESCRIPTION
**Pull request by danielgblanco**
This change adds some clarification to the requirements needed for a project to be approved, and how to represent projects that have not yet been approved in the project board.

### CODE REVIEW

**Review by tedsuo — APPROVED** [review]

**Review by mtwo — APPROVED** [review]

**Comment by danielgblanco** [comment]
I feel like if the language includes "a majority of of GC members" we should probably get a majority of GC members to approve. I'll wait for one more approval to merge.

File: project-management.md
Comment (svrnm) [inline project-management.md:40]: Do we need to vote on projects during GC calls, via approval of the PR or otherwise? I think being even more specific can help. Does majority mean we need 5 approvals or is it lazy voting at some point?

**Review by tigrannajaryan — APPROVED** [review]

File: project-management.md
Comment (svrnm) [reply project-management.md:40]: this is non-blocking

**Review by svrnm — APPROVED** [review]

File: project-management.md
Comment (danielgblanco) [reply project-management.md:40]: Sorry I didn't get to this... I think I'd prefer for it to be 5 approvals on the PR from GC members.

**Pull request merged**
